package.json

```
@@ -16,14 +16,19 @@
     "Other"
   ],
   "activationEvents": [
-    "onCommand:extension.elixirJumpToTest"
+    "onCommand:extension.elixirJumpToTest",
+    "onCommand:extension.elixirRunTestFile"
   ],
   "main": "./extension.js",
   "contributes": {
     "commands": [
       {
         "command": "extension.elixirJumpToTest",
         "title": "Elixir test: Jump"
+      },
+      {
+        "command": "extension.elixirRunTestFile",
+        "title": "Elixir test: run all tests on file"
```

```suggestion
        "title": "Elixir test: Run all tests on file"
```

Just to keep the pattern 😬